### DISCUSSION TITLE
Copy releases from roboll/helmfile to new repo

### DISCUSSION THREAD

**Discussion by ghost** [helmfile/helmfile#205] [Ideas]
In https://github.com/feniix/asdf-helmfile/issues/4#issuecomment-1171357988, @lemeurherve requested that helmfile/helmfile also distribute the old roboll/helmfile releases so that downstream packagers do not have to conditionally switch between the two of them to figure out how to download a release of a specific version.

Should be a pretty simple job with the GitHub API and a bash script, I think. Thoughts?

**Comment by philomory** [comment]
One wrinkle here is that the new repo also packages things in a different format and with different filenames than the old repo. Are you suggesting that the old releases get copied _exactly_ as they are? That would mean that downstream packages may not have to switch repositories based on version, but they'll still have to switch URLs, decide whether or not to extract tarballs, etc., based on version. Once you're already doing that, switching repositories isn't much of an additional burden - that's exactly what I ended up doing in my [patch](https://github.com/mamezou-tech/setup-helmfile/pull/40) to [mamezou-tech/setup-helmfile](https://github.com/mamezou-tech/setup-helmfile).

On the other hand, they _could_ re-package old versions in this repo using the new packaging method (tarballs rather than bare binaries, different file naming patterns); this will be a bit more work, though, I think.

**Reply by ghost** [reply]
Eugh, fair enough. I'll leave it to @lemeurherve to comment on whether this would actually help much - I did already implement exactly what you mentioned in https://github.com/feniix/asdf-helmfile/pull/5 anyway, so I guess this is fixed in downstream.

Obviously it would be _great_ if the existing releases were copied over and repackaged, but that would be a fair bit of work. So, I guess it's up to whether the maintainers think that is a good enough time investment to be worthwhile.
